Resources/ServerInfo/_NF/Guidebook/Shipyard/Ceres.xml

```diff
@@ -0,0 +1,107 @@
+<Document>
+  # CERES-CLASS SERVICE SHUTTLE
+  <Box>
+  <GuideEntityEmbed Entity="KitchenKnife"/>
+  <GuideEntityEmbed Entity="KitchenMicrowave"/>
+  <GuideEntityEmbed Entity="KitchenElectricGrill"/>
+  </Box>
+  [color=#a4885c]Ship Size:[/color] Medium
+
+  [color=#a4885c]Recommended Crew:[/color] 2-4
+
+  [color=#a4885c]Power Gen Type:[/color] Plasma
+
+  [color=#a4885c]Expeditions:[/color] None
+
+  [color=#a4885c]IFF Console:[/color] None
+
+  [color=#a4885c]Available Roles:[/color] Contractor, Pilot, Mercenary
+
+  "A medium-size, high-class restaurant ship with ample seating, integrated botany and a dining room for VIP guests."
+
+  [italic]Brought to you by BlueBird Starship Design & Construction.[/italic]
```

Comment:
Note: This lone `&` is invalid XML, but the parser accepts it. If I write the correct `&amp;` instead, then the guidebook shows the literal text `Design &amp; Construction`. C'est la vie.